test/binCases/config-location/not-found/not-found.test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+const { run } = require("../../../testUtils");
+
+test("not-found", () => {
+	const { code, stdout, stderr } = run(__dirname, [
+		"--output-filename",
+		"[name].js",
+		"--output-chunk-filename",
+		"[id].chunk.js",
+		"--target",
+		"async-node",
+		"--mode",
+		"production",
+	]);
+	expect(code).toBe(0);
+	expect(stdout).toEqual(expect.anything());
+	expect(stdout).toContain("./src/index.js");
+	expect(stderr).toHaveLength(0);
```

Comment:
Apart the `./src/index.js`, is there anything else that we should test in the specific (anything() is too generic)? Also, it is worth to have a snapshot test?

Reply:
`expect(stdout).toEqual(expect.anything());` could be stripped